lib/love-you.json

```diff
@@ -40,7 +40,7 @@
   "chichewa": "Ndimakukonda",
   "chickasaw": "Chiholloli",
   "chinese traditional": "我愛你",
-  "chinese simplified": "我愛你",
+  "chinese simplified": "我爱你",
```

Comment:
Yes! Got it. I misunderstood when I reviewed it. :smile: Landed on the master branch,

:ship: **`1.0.2`** :tada: Thanks!